### ISSUE TITLE
Cannot enter portals without disconnecting the player in dedicated server environments

### ISSUE THREAD

**Issue by Alkiasan** [apace100/origins-fabric#420]
https://gist.github.com/Alkiasan/9136dc6ce1d3d3718823e8f9573f84c1
Running 0.11.6-1.17.1 fabric with the latest version of Origins.

**Comment by Alkiasan** [comment]
Followup to this: For some reason, I'm able to set other's origins, but when they set their own origin, their playerdata breaks and the server log constantly reports "Unable to save (playername)'s playerdata"

**Comment by Alkiasan** [comment]
Followup part 2: This seems to happen when entering an area with entities owned by the person with the broken playerdata, or teleporting to the player themselves.
[crash-2021-08-30_19.35.53-server.txt](https://github.com/apace100/origins-fabric/files/7080161/crash-2021-08-30_19.35.53-server.txt)

**Comment by apace100** [comment]
Do you have any datapacks installed on your server?

**Comment by Alkiasan** [comment]
Yes, mainly vanillatweaks, some dimension datapacks, and various origin datapacks.
Going through portals works now since i've made various changes serverside, Invalid playerdata still seems to happen randomly to players, causing me to have to reset their playerdata for them to join. There's no obvious reason as to why that happens far as I can tell...
My world was also updated from 1.16 to 1.17, so this may be the same issue as #427

**Comment by apace100** [comment]
Did you test the newest version of Origins (1.1.0, released yesterday)?
It fixed many bugs, including (hopefully) the "Invalid Player Data" issue.

**Comment by Alkiasan** [comment]
> Did you test the newest version of Origins (1.1.0, released yesterday)? It fixed many bugs, including (hopefully) the "Invalid Player Data" issue.

I tried to update, but it seems the latest ver is incompatible with Origins: Umbrellas now... Assuming it's something on their side?
Log:
https://gist.github.com/Alkiasan/a8f5ccba0abaedc403745ef7e8bc62e2

**Comment by apace100** [comment]
Yeah, the update broke compatibility with most add-ons, sadly. Will have to wait for them to update.
